zerver/openapi/javascript_examples.js

```diff
@@ -115,11 +115,11 @@ add_example('create_user', '/users:post', 200, async (client) => {
     // {code_example|end}
 });
 
-add_example('get_custom_emoji', '/realm/emoji:get', 200, async (client) => {
+add_example('get_custom_emoji', '/realm/emoji:get', 200, async (client) =>
     // {code_example|start}
-    return await client.emojis.retrieve();
+    await client.emojis.retrieve()
     // {code_example|end}
-});
+);
```

Comment:
Hmm, I'm not sure the arrow-body-style change is good here.  I agree it's a single expression, but the comments mean that this function is now kinda evil.

Reply:
Possibly we should just exclude this file -- those functional comments are the only reason I'd expect it to not be appropriate to put the comment somewhere else.

Reply:
Excluded.